docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1258,6 +1258,44 @@ Renames an existing field. If the field doesn't exist or the new name is already
 }
 --------------------------------------------------
 
+[[script-processor]]
+=== Script Processor
+
+Script Processor allows inline, stored, and file scripts to be executed within Ingest Pipelines.
+
+See <<modules-scripting-using, How to use scripts>> to learn more about writing scripts. Since provided scripts
+are potentially compiled for every document, it is important to understand how script caching
+works. To learn more about caching see <<modules-scripting-using-caching-note, Note on Caching>>
+
+[[script-options]]
+.Script Options
+[options="header"]
+|======
+| Name                   | Required  | Default | Description
+| `field`                | yes       | -       | The field to set
+| `lang`                 | no        | -       | The scripting language
+| `file`                 | no        | -       | The script file to refer to
+| `id`                   | no        | -       | The stored script id to refer to
+| `inline`               | no        | -       | An inline script to be executed
+|======
+
+The current ingest document is provided into the script context using the special `ctx` variable.
```

Comment:
Change to: You can access the current ingest document from within the script context by using the ctx variable.